app/src/main/java/org/mozilla/fenix/settings/logins/SavedLoginsView.kt

```diff
@@ -60,6 +65,7 @@ class SavedLoginsView(
     }
 
     fun update(state: LoginsListState) {
+        // todo MVI views should not have logic. Needs refactoring.
```

Comment:
Let's file an eng health followup for this! 😄

Reply:
Sure!
Filed https://github.com/mozilla-mobile/fenix/issues/12588.